README.md

```diff
@@ -183,95 +207,51 @@ myDialer, err := cloudsqlconn.NewDialer(
 )
 ```
 
-### Using the dialer with database/sql
-
-Using the dialer directly will expose more configuration options. However, it is
-possible to use the dialer with the `database/sql` package.
-
-#### Postgres
+### Using the dialer with pgx
```

Comment:
Do we recommend one of `pgx`? or `database/sql` approach over the other? Or if this examples uses connection pooling and the other doesn't should that be explicitly called out here in the README?

Reply:
I'll make it clear what the recommendation is here.